.polygon/websocket.json

```diff
@@ -312,6 +312,10 @@
                       "type": "integer",
                       "description": "The Timestamp in Unix MS."
                     },
+                    "q": {
```

Comment:
We should update the websocket client for this field as well right?

Reply:
I believe it's already included in EquityQuotes.

Reply:
client returns q;
im not 100% sure how the json file is being used, its being generated via api.polygon.io/specs/websocket.json... the change might not be necessary. Running spec updates on rest and ws just gets it up to date with prod spec.